contracts/base/Escrow.sol

```diff
@@ -169,13 +169,18 @@ contract Escrow is EscrowInterface, TInitializable, BaseUpgradeable, BaseEscrowD
     function repay(uint256 amount) external onlyOwner {
         IERC20 token = IERC20(loans.lendingToken());
```

Comment:
Suggestion: I would rename token to lendingToken to refer that specific token, and replace other occurrences.